<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Startfrom</class>
 <widget class="QWidget" name="Startfrom">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>829</width>
    <height>493</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>829</width>
    <height>493</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/2.jpg);</string>
  </property>
  <widget class="QPushButton" name="pushButton_1">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>230</y>
     <width>131</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 italic 16pt &quot;Adobe Devanagari&quot;;
background-image:url(:/Back0.jpg);
color: rgb(245, 255, 37)</string>
   </property>
   <property name="text">
    <string>Stage One</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>230</y>
     <width>141</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 italic 16pt &quot;Adobe Devanagari&quot;;
background-image: url(:/Back0.jpg);
color:rgb(251, 255, 6)</string>
   </property>
   <property name="text">
    <string>Stage Two</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>230</y>
     <width>141</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 italic 16pt &quot;Adobe Devanagari&quot;;
background-image: url(:/Back0.jpg);
color:rgb(255, 247, 10)</string>
   </property>
   <property name="text">
    <string>Stage Three</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>60</y>
     <width>571</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 242, 56);
</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:28pt; font-weight:600; font-style:italic;&quot;&gt;Forest of Darkness&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
